osutil/cp_linux_test.go

```diff
@@ -36,11 +36,25 @@ func (s *cpSuite) TestCpMulti(c *C) {
 	c.Check(s.f2, testutil.FileEquals, s.data)
 }
 
+func (s *cpSuite) getReadOnlyFd(c *C) *os.File {
+	c.Check(osutil.CopyFile(s.f1, s.f2, osutil.CopyFlagDefault), IsNil)
```

Comment:
(nitpick) it feels a bit unclear why f1->f2 is copied here. All we need is an empty file, so `c.Assert(ioutil.WriteFile(s.f2, nil, 0444)` (if you want to keep the permissions) feels less distracting to me.